roles/kubernetes/node/templates/kubelet.rkt.service.j2

```diff
@@ -62,7 +62,8 @@ ExecStart=/usr/bin/rkt run \
                 $KUBELET_ARGS \
                 $DOCKER_SOCKET \
                 $KUBELET_REGISTER_NODE \
-                $KUBELET_NETWORK_PLUGIN
+                $KUBELET_NETWORK_PLUGIN \
+                $KUBELET_CLOUDPROVIDER
```

Comment:
that's probably a better way of doing it.

Maybe add in `kubelet_start_args=$kubelet_args $docker_socket $kubelet_network_plugin.....` to kubelet.env and have both service files use `$kubelet_start_args`. Want me to add it as part of this PR?